### PR TITLE
fix: 修复 adjust-color label layout 对一些颜色不生效的问题

### PR DESCRIPTION
**Pull request by zqlu**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: src/util/color.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ import colorUtil from '@antv/color-util';
 
 // 根据YIQ亮度判断指定颜色取反色是不是白色
 // http://24ways.org/2010/calculating-color-contrast
-export const isContrastColorWhite = (rgb: string): boolean => {
-  const [r, g, b] = colorUtil.rgb2arr(rgb);
+export const isContrastColorWhite = (color: string): boolean => {
```
Comment (hustcc) [inline src/util/color.ts:5]: 没单侧吧！

Comment (zqlu) [reply src/util/color.ts:5]: done

**Comment by zqlu** [comment]
补上单侧～

**Review by hustcc — APPROVED** [review]

**Comment by hustcc** [comment]
> 补上单侧～

合并，记得关闭一下相关 issue。

**Pull request merged**